github-issue-summary-to-email/main.bal

```diff
@@ -0,0 +1,69 @@
+import ballerina/email;
+import ballerina/log;
+import ballerinax/github;
+
+// Github client configuration parameters
+configurable string githubAccessToken = ?;
+configurable string orgName = ?;
+configurable string repoName = ?;
+
+// Email client configuration parameters
+configurable string smtpPassword = ?;
+configurable string smtpUsername = ?;
+configurable string smtpHost = ?;
+
+// Email configuration parameters 
+configurable string recipientAddress = ?;
+configurable string fromAddress = ?;
+
+public function main() returns error? {
+    string assigneeSummary = "";
```

Comment:
Move the variable declaration to the closest usage.

Reply:
Updated in [462bb79](https://github.com/ballerina-guides/integration-samples/pull/35/commits/462bb7951567e84f94fb0d009f1ff15abfc2d92b)